packages/@aws-cdk/cloudformation-diff/lib/diff-template.ts

```diff
@@ -1,12 +1,14 @@
 // The SDK is only used to reference `DescribeChangeSetOutput`, so the SDK is added as a devDependency.
 // The SDK should not make network calls here
-import type { CloudFormation } from 'aws-sdk';
+import type { DescribeChangeSetOutput as DescribeChangeSet } from '@aws-sdk/client-cloudformation';
 import * as impl from './diff';
 import * as types from './diff/types';
 import { deepEqual, diffKeyedEntities, unionOf } from './diff/util';
 
 export * from './diff/types';
 
+export type DescribeChangeSetOutput = DescribeChangeSet;
```

Comment:
why both this and the import? The change on line 3 lets us reference this type as `DescribeChangeSet`, but without the `as`, you could achieve the same result we get from both lines 3 and 10.

Reply:
This type is used extensively throughout aws-cdk (the cli, not the entire package). This re-export is a workaround In order to not cause a conflict of types in that package. You could argue that we should just migrate that whole package, but that's a much more complex task that we have not yet prioritized.